website/content/api-docs/secret/identity/entity.mdx

```diff
@@ -396,14 +396,17 @@ $ curl \
 ## Merge Entities
 
 This endpoint merges many entities into one entity. Additionally, all groups associated with `from_entity_ids` are merged with those of `to_entity_id`.
+Note that if these entities contain aliases sharing the same mount accessor, the merge will fail unless `conflicting_alias_ids_to_keep` is present, and
+entities must be merged one at a time. This is because each entity can only have one alias with each mount accessor - for more
+information, see the [identity concepts page](/docs/concepts/identity).
 
 | Method | Path                     |
 | :----- | :----------------------- |
 | `POST` | `/identity/entity/merge` |
 
 ### Parameters
 
-- `from_entity_ids` `(array: <required>)` - Entity IDs which needs to get
+- `from_entity_ids` `(list of strings: <required>)` - Entity IDs which need to get
```

Comment:
This was updated as it was the only list of strings on the page to use "array" in lieu of "list of strings". In other words, this change keeps the language consistent throughout the page.